buildout.cfg

```diff
@@ -7,9 +7,9 @@ parts =
   i18n
 github = git://github.com/zopefoundation
 github_push = git@github.com:zopefoundation
-extensions = mr.developer
-auto-checkout =
-  z3c.recipe.i18n
+
+[versions]
+z3c.recipe.i18n = 1.0.0
```

Comment:
I'm not too keen on pinning exact versions: we want to know if newer z3c.recipe.i18n releases somehow break us.

If you need to specify a minimum, buildout lets you say `zc.recipe.i18n = >= 1.0.0`.

Reply:
will do.